docs/editors/debug-adapter-protocol.md

```diff
@@ -18,18 +23,51 @@ activates a code lens. The difference between them is that the former ignores
 all breakpoints being set while the latter respects them. The procedure of
 starting the run/debug session is as follows:
 
-1. Request the debug adapter uri from metals server using the
-   [`debug-adapter-start`](https://github.com/scalameta/metals/blob/master/metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala#L95)
-   command.
-2. Connect the debug adapter extension specific to you editor using the
-   aforementioned uri and let it drive the run/debug session. For reference,
-   take a look at the
-   [vscode implementation](https://github.com/scalameta/metals-vscode/blob/master/src/scalaDebugger.ts)
-   and how it is
-   [wired up together](https://github.com/scalameta/metals-vscode/blob/master/src/extension.ts#L356)
+Then we can request the debug adapter uri from metals server using the
+[`debug-adapter-start`](https://github.com/scalameta/metals/blob/master/metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala#L95)
+command.
+
+### Via simple commands
+
+Apart from using code lenses, users can start a debug session by executing the
+`debug-adapter-start` command with the following params:
+
+- for main class
+
+```json
+{
+  "mainClass": "com.foo.App",
+  "buildTarget": "foo",
+  "args": ["bar"]
+}
+```
+
+- for test class
+
+```json
+{
+  "testClass": "com.foo.FooSuite",
+  "buildTarget": "foo"
+}
+```
+
+`buildTarget` is an optional parameter, which might be useful if there are identical
```

Comment:
```suggestion
`buildTarget` is an optional parameter, which might be useful if they are identical
```

Reply:
I think this one should stay:
`there are identical class names in different modules`
would make the same sense as:
`they are identical class names in different modules`

Reply:
Ahh yea. Good call. Sorry read it too quickly!

Reply:
Maybe `there are identically named classes` ?

Reply:
I think the way you have it is good :)